screens/ChatIndexScreen.tsx

```diff
@@ -0,0 +1,82 @@
+import { View, ScrollView, StyleSheet } from 'react-native';
+import React, { useEffect, useState } from 'react';
+import { RootStackParamList } from '../types';
+import { NativeStackNavigationProp } from '@react-navigation/native-stack';
+
+import TopBar from '../components/TopBar';
+import Chat from '../components/Chat';
+
+import ChatType from '../types/Chat';
+import getChats from '../lib/getChats';
+import NavBar from '../components/NavBar';
+import Notification from '../components/Notification';
+
+type ProfileScreenNavigationProp = NativeStackNavigationProp<
+  RootStackParamList,
+  'ChatIndexScreen'
+>;
+
+export interface ChatIndexScreenProps {
+  navigation: ProfileScreenNavigationProp;
+}
+
+export default function ChatIndexScreen({
+  navigation: { navigate },
+}: ChatIndexScreenProps) {
+  const [chats, setChats] = useState<ChatType[] | null>(null);
```

Comment:
Again, shouldn't be null ever. Just initialize it with an empty array
```suggestion
  const [chats, setChats] = useState<ChatType[]>([]);
```